kofta/public/locales/el-GR/translation.json

```diff
@@ -202,7 +202,7 @@
       }
     },
     "roomChat": {
-      "title": "Τσατ",
+      "title": "Chat",
```

Comment:
Again, not sure why this is being changed back to english.

Reply:
yeah lol